### DISCUSSION TITLE
Export base ? [Jun 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2474] [Discord Threads]
> 🙋‍♂️ **cirsten**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You mean exporting metadata of the base ?  
#  
> 💁 **o1lab**  
#### if so, yes - we will bring it back  
#  
> 💁 **o1lab**  
#### if you meant just the base, we didnt support it anytime  
#  
> 🙋‍♂️ **cirsten**  
#### Must have been metadata  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/986505594200555550)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
